tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/CloudflowStatus.scala

```diff
@@ -102,8 +102,10 @@ object CloudflowStatus {
       nrOfContainers = nrOfContainers)
   }
 
-  private def getStatusFromContainerStates(containerStates: List[ContainerState], nrOfContainers: Int): String =
-    if (containerStates.nonEmpty) {
+  private def getStatusFromContainerStates(containerStates: List[ContainerState], nrOfContainers: Int): String = {
+    if (nrOfContainers == 0) {
```

Comment:
We would prefer to show as pending? not sure about this change.

Reply:
reverted